build_test/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 1.4.0-dev
+
+- Migrate to null safety
```

Comment:
Just for completeness, I want to point out that this package is not fully migrated yet (and shouldn't be released as-is). We're still blocked by `build_resolvers`, which is blocked by `graphs`.

Reply:
Understood, did you plan to look at those or should one of us (either is fine, just want to not duplicate work :) )

Reply:
One of you would have to take a look at https://github.com/dart-lang/graphs/pull/52. As soon as that package is migrated I'm happy to work on `build_resolvers` and the rest of `build_test`.